src/consts/permissions.ts

```diff
@@ -178,6 +178,14 @@ enum PERMISSIONS {
   ConsultationUpdate = 'consultation_update',
   ConstulatationDelete = 'consultation_delete',
   ConsultationRead = 'consultation_read',
+
+  /* -------- EscolaLMS/Stationary-Events -------- */
+
+  StationaryEventsList = 'stationary-event_list',
```

Comment:
plural doesn't match the formula should be StationaryEventList / StationaryEventCreate etc.

Reply:
@mako321 🆙 ?